website/docs/reference/datasources/querying-google-sheets.md

```diff
@@ -1,583 +1,272 @@
-| [**Delete One**](./querying-google-sheets.md#delete-one)            | Use it to delete a spreadsheet or a sheet or row in a sheet                               | Spreadsheet/Sheet/ Sheet Rows |
-| [**Fetch Many**](./querying-google-sheets.md#fetch-many-sheet-rows) | Use it to fetch data from a sheet or all the spreadsheets available in your account.      | Spreadsheet/Sheet Rows        |
-| [**Insert Many**](./querying-google-sheets.md#insert-many)          | Use it to insert multiple rows in a given sheet                                           | Sheet Rows                    |
-| [**Update Many**](./querying-google-sheets.md#update-many)        | Use it to update multiple rows in a given sheet                                           | Sheet Rows                    |
-
-## Fetch details
-
-The operation allows you to fetch the details of a given spreadsheet.
-
-<VideoEmbed host="youtube" videoId="elAabPXK_xk" title="Fetch Details of a Spreadsheet" caption="Fetch Details of a Spreadsheet"/>
-
-You can set the parameter values as below:
+<figure>
+  <img src="/img/google-sheets-query-page.png" style={{width: "100%", height: "auto"}} alt="Configuring a query from the query screen."/>
+  <figcaption align="center"><i>Configuring a query from the query screen.</i></figcaption>
+</figure>
 
-* [Operation](./querying-google-sheets.md#operation) - Fetch details
-* [Entity](./querying-google-sheets.md#entity) - Spreadsheet
-* [Spreadsheet](./querying-google-sheets.md#spreadsheet) - Either select the name of the spreadsheet from the available options, or you can enable the JS label and add the URL of the desired spreadsheet
+You can create [queries](https://docs.appsmith.com/core-concepts/data-access-and-binding/querying-a-database/query-settings) to fetch or write data to your spreadsheets by clicking (**+**) next to **Queries/JS** in the **Explorer** tab and selecting your Google Sheets datasource. You'll be brought to a new screen to set up your query.
 
-:::info
-The **Fetch Details** operation is only available to fetch the details of a spreadsheet.
-:::
-
-## Insert one
+### Operations
 
-With Insert One, you can choose to insert a new row in an existing spreadsheet or add a new spreadsheet.
+**Operation** sets the type of action you want to perform with your query. A "Spreadsheet" is a document, a "Sheet" is a page of a spreadsheet, and "Sheet Rows" are horizontal records in a sheet.
 
-### Insert a new spreadsheet
+| **Operation**                        | **Description**                           | **Available entities:**       |
+| ------------------------------------ | ----------------------------------------- | ----------------------------- |
+| [**Fetch Details**](#fetch-details)  | Fetches metadata about a spreadsheet. | [Spreadsheet](#fetch-details-spreadsheet)  |
```

Comment:
Mention this is only accessible if you give Appsmith the read-write All permission